### PR TITLE
Adds images for the tutorial docs

### PR DESCRIPTION
**Pull request by mpgussert**
# Description

I added images for the tutorials and modified the docs to include them

## Checklist

- [X] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./isaaclab.sh --format`
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have updated the changelog and the corresponding version in the extension's `config/extension.toml` file
- [X] I have added my name to the `CONTRIBUTORS.md` or my name already exists there

### CODE REVIEW

**Review by Mayankm96 — COMMENTED** [review]
Thanks for taking a dig at this. Would suggest arranging them in the same order/name as the tutorial rst itself. Also would be better to have them in JPG format as that usually has smaller file fize.

Also for figures, please make sure that the essential directives are added (for instance, alt and width).

> https://docutils.sourceforge.io/docs/ref/rst/directives.html#figure

File: docs/source/tutorials/03_envs/create_manager_base_env.rst [hunk 1]
```diff
@@ -186,6 +186,7 @@ operations.
 The Code Execution
 ~~~~~~~~~~~~~~~~~~
 
+
```
Comment (Mayankm96) [inline docs/source/tutorials/03_envs/create_manager_base_env.rst]: Please remove these stray enters everywhere.

File: docs/source/_static/tutorials/tutorial_create_direct_workflow.png
Comment (Mayankm96) [inline docs/source/_static/tutorials/tutorial_create_direct_workflow.png:1]: Not sure if this view is what pop ups by default or this was chosen intentionally. Either way the camera view should be adjusted to not show the white background (looks ugly).

Same goes for other images.

**Comment by mpgussert** [comment]
> Thanks for taking a dig at this. Would suggest arranging them in the same order/name as the tutorial rst itself. Also would be better to have them in JPG format as that usually has smaller file fize.
> 
> Also for figures, please make sure that the essential directives are added (for instance, alt and width).
> 
> > https://docutils.sourceforge.io/docs/ref/rst/directives.html#figure

I converted all the images to jpg, but i'm not sure what you mean about these directives

**Review by kellyguo11 — APPROVED** [review]

**Pull request merged**